app/labor/markdown_parser.rb

```diff
@@ -18,10 +18,10 @@ def finalize(link_attributes: {})
     sanitized_content = sanitize_rendered_markdown(html)
     begin
       parsed_liquid = Liquid::Template.parse(sanitized_content)
-    rescue StandardError => e
-      raise StandardError, e.message
```

Comment:
This Standard error does get caught here https://github.com/thepracticaldev/dev.to/blob/master/app/models/article.rb#L434. Doesn't that stops it from being propagated to our error reporting?

Reply:
Doesn't look like it does in all cases
```
Liquid::SyntaxError: Liquid syntax error: Variable '{{' was not properly terminated with regexp: /\}\}/
markdown_parser.rb  96 tags_used(...)

[PROJECT_ROOT]/app/labor/markdown_parser.rb:96:in `tags_used'
94     cleaned_parsed = escape_liquid_tags_in_codeblock(@content)
95     tags = []
96     Liquid::Template.parse(cleaned_parsed).root.nodelist.each do |node|
97       tags << node.class if node.class.superclass.to_s == LiquidTagBase.to_s
98     end

comment.rb  157 custom_css(...)
[PROJECT_ROOT]/app/models/comment.rb:157:in `custom_css'

comments_controller.rb  72 create(...)
[PROJECT_ROOT]/app/controllers/comments_controller.rb:72:in `create'
```

Reply:
Thanks! I made a mistake there, this is a comment related error and not article.